### ISSUE TITLE
eslint crash with plugin:vue-pug/vue3-recommended

### ISSUE THREAD

**Issue by capacic** [rashfael/eslint-plugin-vue-pug#19]
Eslint work when I dont't have 'plugin:vue-pug/vue3-recommended' in eslint config, but when I add this node crashes.

```js
	extends: [
		'eslint:recommended',
		'plugin:codeceptjs/recommended',
		'plugin:json/recommended', 
		'plugin:vue/vue3-recommended',
		'plugin:vue-pug/vue3-recommended'
	]
```

Run: 

```
npx eslint '**/*.{js,vue,json,html,xml}'

#
# Fatal error in , line 0
# Fatal JavaScript invalid size error 169220804
#
#
#
#FailureMessage Object: 0x16baf0048
 1: 0x104417534 node::NodePlatform::GetStackTracePrinter()::$_3::__invoke() [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
 2: 0x1053f3fc0 V8_Fatal(char const*, ...) [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
 3: 0x1046a1c08 v8::internal::FactoryBase<v8::internal::Factory>::NewFixedArrayWithFiller(v8::internal::Handle<v8::internal::Map>, int, v8::internal::Handle<v8::internal::Oddball>, v8::internal::AllocationType) [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
 4: 0x1048364f0 v8::internal::(anonymous namespace)::ElementsAccessorBase<v8::internal::(anonymous namespace)::FastPackedObjectElementsAccessor, v8::internal::(anonymous namespace)::ElementsKindTraits<(v8::internal::ElementsKind)2>>::GrowCapacity(v8::internal::Handle<v8::internal::JSObject>, unsigned int) [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
 5: 0x104a5c37c v8::internal::Runtime_GrowArrayElements(int, unsigned long*, v8::internal::Isolate*) [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
 6: 0x104df72ac Builtins_CEntry_Return1_DontSaveFPRegs_ArgvOnStack_NoBuiltinExit [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
 7: 0x13076c1a0
 8: 0x13060aa4c
 9: 0x130727c68
10: 0x1305f6134
11: 0x1305f64c0
12: 0x1305f64c0
13: 0x13056f74c
14: 0x1306ad3c4
15: 0x1303b1848
16: 0x13038e2c0
17: 0x1304af154
18: 0x104e0ef0c Builtins_ArrayForEach [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
19: 0x130454400
20: 0x1304a774c
21: 0x1304562b0
22: 0x130409c14
23: 0x13074b7ec
24: 0x104e186fc Builtins_ArrayMap [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
25: 0x1306e0900
26: 0x130495a48
27: 0x13058b068
28: 0x1304627c8
29: 0x130456784
30: 0x1305619f4
31: 0x104d74064 Builtins_InterpreterEntryTrampoline [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
32: 0x104d74064 Builtins_InterpreterEntryTrampoline [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
33: 0x104daa8b4 Builtins_AsyncFunctionAwaitResolveClosure [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
34: 0x104e4ae38 Builtins_PromiseFulfillReactionJob [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
35: 0x104d9a834 Builtins_RunMicrotasks [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
36: 0x104d723c4 Builtins_JSRunMicrotasksEntry [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
37: 0x104649f40 v8::internal::(anonymous namespace)::Invoke(v8::internal::Isolate*, v8::internal::(anonymous namespace)::InvokeParams const&) [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
38: 0x10464a430 v8::internal::(anonymous namespace)::InvokeWithTryCatch(v8::internal::Isolate*, v8::internal::(anonymous namespace)::InvokeParams const&) [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
39: 0x10464a60c v8::internal::Execution::TryRunMicrotasks(v8::internal::Isolate*, v8::internal::MicrotaskQueue*) [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
40: 0x104672584 v8::internal::MicrotaskQueue::RunMicrotasks(v8::internal::Isolate*) [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
41: 0x104672e20 v8::internal::MicrotaskQueue::PerformCheckpoint(v8::Isolate*) [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
42: 0x104304c5c node::InternalCallbackScope::Close() [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
43: 0x104304fe8 node::InternalMakeCallback(node::Environment*, v8::Local<v8::Object>, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*, node::async_context) [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
44: 0x10431aea8 node::AsyncWrap::MakeCallback(v8::Local<v8::Function>, int, v8::Local<v8::Value>*) [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
45: 0x1043bc51c node::fs::FSReqCallback::Reject(v8::Local<v8::Value>) [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
46: 0x1043bcd9c node::fs::FSReqAfterScope::Reject(uv_fs_s*) [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
47: 0x1043bd1d0 node::fs::AfterStat(uv_fs_s*) [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
48: 0x1043b38e8 node::MakeLibuvRequestCallback<uv_fs_s, void (*)(uv_fs_s*)>::Wrapper(uv_fs_s*) [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
49: 0x104d509a4 uv__work_done [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
50: 0x104d5414c uv__async_io [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
51: 0x104d668f0 uv__io_poll [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
52: 0x104d5461c uv_run [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
53: 0x104305704 node::SpinEventLoopInternal(node::Environment*) [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
54: 0x1043f4bb0 node::NodeMainInstance::Run() [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
55: 0x1043831d4 node::LoadSnapshotDataAndRun(node::SnapshotData const**, node::InitializationResultImpl const*) [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
56: 0x10438343c node::Start(int, char**) [/Users/user/.nvm/versions/node/v19.7.0/bin/node]
57: 0x1a6f43f28 start [/usr/lib/dyld]
 ELIFECYCLE  Command failed.
[1]    63308 trace trap  pnpm lint
```

**Comment by rashfael** [comment]
I need to know the source code which makes the linter behave this way to be able to help you.

**Comment by rashfael** [comment]
Can reproduce. Seems to be an issue with `vue/html-indent`. I will investigate further.

**Comment by rashfael** [comment]
After looking into it more and looking into adding unit tests I'm inclined to turn off `vue/html-intent` for pug.
The things it tries to check are mainly irrelevant for pug, since the pug parser already checks things like mixed indent and a lot of things that can go wrong with indentation are just not possible with pug.

Things like
```pug
p(
  foo="foo"
    bar="bar"
)
```
also seem not to be detected by `vue/html-intent` so I need to create a custom rule for pug indent linting.

**Comment by rashfael** [comment]
fixed in `0.6.0`.
